typings.json

```diff
@@ -2,6 +2,6 @@
   "globalDependencies": {
     "assertion-error": "registry:dt/assertion-error#1.0.0+20160316155526",
     "jasmine": "registry:dt/jasmine#2.2.0+20160505161446",
-    "node": "registry:dt/node#6.0.0+20160514165920"
+    "node": "registry:env/node#6.0.0+20160902022231"
   }
 }
```

Comment:
With typescript 2, we no longer need the typings file or the dependency on the typings package.

Community type definitions are now stored in npm, under the @types namespace.

So we can get rid of the `npm run typings` script and instead do a one-time

`
npm install --save @types/jasmine @types/node @types/assertion-error
`